### PR TITLE
Add failing ContentFrame test

### PR DESCRIPTION
**Pull request by Androbin**
See issue #1945

### CODE REVIEW

File: lib/PuppeteerSharp.Tests/ElementHandleTests/ContentFrameTests.cs [hunk 1]
```diff
@@ -24,5 +28,17 @@ public async Task ShouldWork()
             var frame = await elementHandle.ContentFrameAsync();
             Assert.Equal(Page.FirstChildFrame(), frame);
         }
+
+        [PuppeteerTest("elementhandle.spec.ts", "ElementHandle.contentFrame", "should work headful")]
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/ElementHandleTests/ContentFrameTests.cs:32]: We don't need this if it's not puppeteer.

```suggestion
```

**Comment by Androbin** [comment]
It looks like the frame gets detached for no reason

**Comment by kblok** [comment]
Can you see if you get the same in puppeteer in node?

**Comment by Androbin** [comment]
Node works like a charm:
```js
require('puppeteer').launch({ headless: false }).then(async browser => {
  const page = await browser.newPage()
  await page.goto('http://localhost:8081/frame-example.html')
  const elementHandle = await page.$('iframe')
  const frame = await elementHandle.contentFrame()
  const content = await frame.content()
  console.log(content)
  await browser.close()
})
```

**Comment by kblok** [comment]
Good catch @Androbin

**Comment by kblok** [comment]
Fixed on https://github.com/hardkoded/puppeteer-sharp/pull/1979

**Pull request merged**